docs/datasets_and_tables/transitdatabase.md

```diff
@@ -2,14 +2,52 @@
 
 ### Transit Stacks
 
+| **Name**<br>*Key(s)*| **Description** |
+| :------------- |  :-------------- |
+| `organizations` | See above for definition.  Within Transit Stacks context, each organization MAY: <ul><li>*sell* one more `products`</li><li>*own* one or more `contracts` which license the use of one or more `products` to one or more `organizations`.</li><li>*hold* one or more `contracts` with another `organization` to use one or more `products` with a `service` that they *manage* or *operate*.</li><li>*consume* one or more `gtfs datasets`</li><li>*produce* one or more `gtfs datasets`</li><li>*publish* one or more `gtfs datsets`</li><li>*use* one or more `products` as a `organization stack component` for a `service` that they *manage* or *operate*</li></ul>|
```

Comment:
I recommend dropping `gtfs` prefix from datasets to be agnostic to dataset type (could include GBFS in future for example):

```suggestion
| `organizations` | See above for definition.  Within Transit Stacks context, each organization MAY: <ul><li>*sell* one more `products`</li><li>*own* one or more `contracts` which license the use of one or more `products` to one or more `organizations`.</li><li>*hold* one or more `contracts` with another `organization` to use one or more `products` with a `service` that they *manage* or *operate*.</li><li>*consume* one or more `datasets`</li><li>*produce* one or more `datasets`</li><li>*publish* one or more `datsets`</li><li>*use* one or more `products` as a `organization stack component` for a `service` that they *manage* or *operate*</li></ul>|
```